relay-dynamic-config/src/global.rs

```diff
@@ -46,6 +55,21 @@ impl GlobalConfig {
             Ok(None)
         }
     }
+
+    /// Returns the generic inbound filters.
+    pub fn filters(&self) -> Option<&GenericFiltersConfig> {
+        match &self.filters {
+            ErrorBoundary::Err(_) => None,
```

Comment:
In this case, we filter nothing, right?

Not a strong opinion on this, because we can always bump the metrics extraction version, but Isn't the following scenario still relevant:

1. We make a breaking change to the filter config.
2. Outdated external relays stop filtering, and extract metrics from everything that was previously filtered out.

I'm also on board with just deciding that we'll never make a breaking change, but in that case, why do we need the error boundary?

Reply:
Should be addressed in [bfb8657](https://github.com/getsentry/relay/pull/3161/commits/bfb8657364cfc5fa9739b625bf72f78aec987d20).